substrate/primitives/runtime/src/generic/unchecked_extrinsic.rs

```diff
@@ -83,7 +83,7 @@ pub enum Preamble<Address, Signature, Extension> {
 	Bare(ExtrinsicVersion),
 	/// An old-school transaction extrinsic which includes a signature of some hard-coded crypto.
 	/// Available only on extrinsic version 4.
-	Signed(Address, Signature, ExtensionVersion, Extension),
```

Comment:
Will this require changes in the signers?

Reply:
no the preamble encoding and decoding is manually written, and the extension version wasn't part of the encoded value, instead 0 was always used.

Reply:
Yes, it's a completely transparent change for everyone.